Client/Frontend/Contextual Hint/ContextualHintViewModel.swift

```diff
@@ -19,6 +19,10 @@ enum ContextualHintType: String {
     case toolbarLocation = "ToolbarLocation"
 }
 
+enum ContextualHintCopyType {
+    case action, description
+}
```

Comment:
Would a better place for this be the copy provider file as it's more directly related to that?